dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/DataIntegrity.ts

```diff
@@ -202,11 +203,6 @@ describeIf(
           testData.companyInformation,
           testData.t
         );
-        checkCreditInstitutionValues(testData.t, true, true);
-
-        checkCommonFields("AssetManagement", testData.t.eligibilityKpis!.AssetManagement);
-
-        checkInsuranceValues(testData.t);
       }
     );
```

Comment:
Please merge main into this branch and resolve them if there are any conflicts. I think it is best if you do it as you have the best overview.
If there are any strange merge conflicts with main that you are not sure about, let's discuss it.

Reply:
Please assure that the fake fixtures are also updated by merging main into this branch.
This point is important, because the `reference` field in the `referencedReports` of the fake fixtures still contains file-names instead of hashes.

Reply:
I will take care of this since I need the merge right now to continue dev